systemtest/src/test/java/io/strimzi/systemtest/BaseST.java

```diff
@@ -510,7 +510,7 @@ protected void checkReadinessLivenessProbe(String podNamePrefix, String containe
         }
```

Comment:
I wonder if this should actually be called `AbstractST`
To match with the naming conventions we've gone elsewhere in Strimzi, i.e. AbstractModel